plugins/outputs/timestream/sample.conf

```diff
@@ -47,6 +47,26 @@
   ## Specifies if the plugin should create the table, if the table does not exist.
   create_table_if_not_exists = true
 
+  ## Specifies if table should be created with composite partition key
+  ## that determines how your data will be stored and distributed.
+  ## create_table_composite_partition_key is an array of array with type string
+  ## Dimension type partition key requires 3 elements:
+  ##    1. "Dimension" to indicate its type
+  ##    2. Your composite_partition_key_dimension_name
+  ##    3. Enforcement level can be "true" or "false". If set to true dimension key must be specified
+  ##        during ingestion of new records or else they will be rejected. If set to false then records
+  ##        without specifying the dimension key can still be ingested.
+  ##    example: ["Dimension", "composite_partition_key_dimension_name", "true"]
+  ## Measure type partition key requires 1 element:
+  ##    1. "Measure_name" to indicate its type
+  ##    example: ["Measure_name"]
+  ##
+  ## NOTES:
+  ## You can't add or remove a partition key after the table has been created.
+  ## We currently only support using one partition key.
```

Comment:
Is the We telegraf itself or AWS Timestream?

Reply:
Updated the we to Timestream.